cricPredict/User_profile/views.py

```diff
@@ -47,11 +99,11 @@ def form_valid(self, form):
             return HttpResponse("Invalid Username or Password")
 
     def form_invalid(self, form):
+        print(form.errors.as_json())
         return render(self.request, 'login.html', {'form': form})
 
 
 class Logout(RedirectView):
-
     def get(self, request1, *args, **kwargs):
         logout(request1)
         return redirect("/")
```

Comment:
same here, Django's redirection shouldn't work because we aren't relying on Django to render the templates